### PR TITLE
Foldure structure

### PR DESCRIPTION
**Pull request by nerijusabromavicius**

### CODE REVIEW

File: src/components/App/App.css [hunk 1]
```diff
@@ -0,0 +1,5 @@
+.App {
+  text-align: center;
+  top: 5px;
+}
+
```
Comment (gabriuz) [inline src/components/App/App.css:5]: embeddinciau i app.js

File: src/components/App/App.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+export default class App extends React.Component {
+  render() {
+    return (
+      <div>
+        <Header />
+      </div> 
+    );
+  }
+}
```
Comment (gabriuz) [inline src/components/App/App.js:12]: embeddinus .css liktu vienintelis App.js - tada nereiktu taip giliai nestint failo - butu `src/componenents/App.js`

File: src/components/Header/Header.css [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  .App-header {
+    background-color: #222;
+    height: 80px;
+    padding: 6px;
+    color: white;
+  }
+  
+  .App-intro {
+    font-size: large;
+  }
```
Comment (gabriuz) [inline src/components/Header/Header.css:20]: same shit kaip auksciau, plius indikuoja `no newline at end of file` - reiktu pasidomet `.editorconfig` arba `.gitattributes` failiukais

File: src/index.js [hunk 4]
```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
-import App from './App';
+import App from './components/App/App';
```
Comment (gabriuz) [inline src/index.js:4]: nenestinant butu `import App from './components/App.js';`

**Review by gabriuz — CHANGES_REQUESTED** [review]

File: src/components/Header/Header.css (hunk 3 above)
Comment (Verrrtigo) [reply src/components/Header/Header.css:20]: https://help.github.com/articles/dealing-with-line-endings/

**Pull request merged**